src/mod/server.mod/servmsg.c

```diff
@@ -1148,6 +1152,47 @@ static int got465(char *from, char *msg)
   return 1;
 }
 
+static int got410(char *from, char *msg) {
+  char *cmd;
+
+  newsplit(&msg);
+ 
+  putlog(LOG_SERV, "*", "%s", msg);
+  cmd = newsplit(&msg);
+  putlog(LOG_MISC, "*", "CAP sub-command %s not supported", cmd);
+
+  return 1;
+}
+
+static int got421(char *from, char *msg) {
+  newsplit(&msg);
+  putlog(LOG_SERV, "*", "%s reported an error: %s", from, msg);
+
+  return 1;
+}
+
+static int gotcap(char *from, char *msg) {
```

Comment:
eggdrop code style is: "{" at new line. not only here, but other functions also.

Reply:
We don't have an established coding guide... maybe we should, but it sounds like a lot of work :P I think we can make a case either way on this